client/packages/core/src/Reactor.js

```diff
@@ -24,8 +24,9 @@ const STATUS = {
   ERRORED: "errored",
 };
 
-const QUERY_ONCE_TIMEOUT = 10_000;
+const QUERY_ONCE_TIMEOUT = 30_000;
```

Comment:
Won't we time out on the backend after 5 seconds anyway?

Reply:
Yes! My thinking is that this timeout will help in cases like: `the user kicks off queryOnce, goes offline for 25 seconds, etc` 

I do think we may want to increase backend timeout too; I am just not 100% sure how our receive-workers will take it; I worry we'll end up saturating the pool. (Perhaps we can look into making an infinite pool of virtual threads, but that can make debugging more difficult)